### ISSUE TITLE
feat: Add GridView Support

### ISSUE THREAD

**Issue by hintbox** [VeryGoodOpenSource/very_good_infinite_list#79]
### Description

Great library, could you support also gridview for infinite listing

### Requirements

- [ ] All CI/CD checks are passing.
- [ ] There is no drop in the test coverage percentage.

**Comment by tomarra** [comment]
Hi @hintbox 👋  thanks for opening this issue!

As of right now we don't have a use case that would drive this need for us so I'm going to mark this as a P3 issue. That being said if it is something that you feel you can help implement it would be great to start to detail out what the feature would look like both code architecture wise as well as UI implementation. With that documented here you (or anyone else) could then start down the path of opening a PR to get it reviewed and merged in the future.

**Comment by shivams1110** [comment]
@hintbox I also need Gridview, So I forked this project and added GridView functionality.

Clone this **https://github.com/shivams1110/very_good_infinite_list** into your local system and use it.

Use this **InfiniteGrid** widget.

and let me know if it is helpful.

**Comment by tomarra** [comment]
@shivams1110 Any interest in opening a PR to contribute that back?

**Comment by shivams1110** [comment]
> @shivams1110 Any interest in opening a PR to contribute that back?

Yeah, I am interested But for that, I need to write a test case for this implementation, and after that, I will do it.

**Comment by keiwanmosaddegh** [comment]
@shivams1110 amazing contribution, I'd love to see this go all the way! 🤩 Let me know if I can help!

**Comment by shivams1110** [comment]
> @shivams1110 amazing contribution, I'd love to see this go all the way! 🤩 Let me know if I can help!

Sure please check this pr once, if you can help me in this.

**Comment by hintbox** [comment]
hi, any updates

**Comment by rlee1990** [comment]
Hi any update on this?
